src/__swaps__/screens/Swap/hooks/useSwapNavigation.ts

```diff
@@ -66,7 +66,7 @@ export function useSwapNavigation({
     'worklet';
```

Comment:
worklets are automatically memoized under the hood so useCallback is unnecessary fyi